_activities/gsoc.md

```diff
@@ -49,100 +49,100 @@ For further information, feel free to contact the HSF GSoC admins or join our op
 
 ### Student pages
 
-The CERN-HSF 2022 student blogs are now available. [Check them out!]({{ site.baseurl }}/gsoc/2022/blogs.html)
+The CERN-HSF 2023 student blogs are now available. [Check them out!]({{ site.baseurl }}/gsoc/2023/blogs.html)
```

Comment:
There is only one blog post from 2023, so maybe we need to remove this link? I am not sure it gives a great impression.